pkg/attacher/bcc_attacher.go

```diff
@@ -62,8 +62,14 @@ var (
 	EnableCPUFreq = true
 )
 
-func loadModule(objProg []byte, options []string) (*bpf.Module, error) {
-	m := bpf.NewModule(string(objProg), options)
+func loadModule(objProg []byte, options []string) (m *bpf.Module, err error) {
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
there is no more panic or fatalf, recover() won't triggered

Reply:
well, the 1st panic real happens at loadModule function, and the `m := bpf.NewModule(string(objProg), options)` real good as it init variable of `m`. As a hot fix, I do agree to make it as it is.
or maybe we'd better give some comments in line around or put init variable of m in creation/init function when controller been created.

Reply:
@rootfs Sorry I did not understand your comment...
We do have a panic happening in the case there is any problem with bpf attach
But Kepler should not stop to run if we don't have bpf